src/pybind/mgr/balancer/module.py

```diff
@@ -226,6 +242,9 @@ class Module(MgrModule):
     run = True
     plans = {}
     mode = ''
+    key = 'pgs'
```

Comment:
It has been referenced in 
`do_crush_compat` function at [here](https://github.com/liewegas/ceph/pull/60/files#diff-aaf33d75795c314c9b5232fb40490fb7R661).